### DISCUSSION TITLE
[TextBlock][Inlines] Avoid space between runs

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#14279] [Q&A] [answered]
Considerig following Xaml

```xaml
  <TextBlock TextAlignment="Left" Margin="5" FontFamily="Segoe UI">
    <TextBlock.Inlines>
      <Run Text="[Sea]" FontWeight="Bold"/>
      <Run Text="led"/>
    </TextBlock.Inlines>
  </TextBlock>
``` 

It displayed as:

![immagine](https://github.com/AvaloniaUI/Avalonia/assets/12531229/cf0e8bd6-f556-4889-aab3-ae80e5e5a856)

Is there a way to avoid the space between `[Sea]` and `led`?

I try with `Avalonia 11.0.7` and c3911177956897d7a5176f9d59932b308aebc0c9

**Comment by Gillibald** [comment, marked as answer]
Line feeds are normalized to one space. You can only avoid this by writing all tags at one line.

**Reply by workgroupengineering** [reply]
Thank you very much, it works great.

**Reply by workgroupengineering** [reply]
@Gillibald Is there a way to change behavior? Every time I format the xaml I am forced to revert the inlines.

**Reply by MihaMarkic** [reply]
@Gillibald Sorry for being dense by what do you mean by 'writing all tags at one line'? And why the heck is that separator value fixed...

**Reply by Gillibald** [reply]
https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/xaml-and-whitespace

**Reply by MihaMarkic** [reply]
Ha, thanks, I'm slow today :)
However, that's not a perfect solution IMO. Linefeed normalization should be settable per TextBlock (or InlinesHost). Perhaps the worst problem is auto formatting of XAML - like with Rider, it won't respect those tags in single line I think. I have to also check with Jetbrains folk.
